Author: aszlig <aszlig@nix.build>
Date:   Sat Oct 13 02:40:20 2018 +0200

    migration-runner: Don't expect a Promise
    
    The migration runner got refactored a bit[1] and the processUsers
    function now is expected to return a Promise. While this works for newer
    migrations, this will fail for older migration scripts.
    
    For now, I'm essentially reverting this change until we have a better
    solution in place to handle both the old and new way of running
    migrations.
    
    [1]: https://github.com/HabitRPG/habitica/commit/70192e493568a6aebca926d
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: revert-migration-runner-promise.patch

diff --git a/migrations/migration-runner.js b/migrations/migration-runner.js
index c607b5eac..92c5855b8 100644
--- a/migrations/migration-runner.js
+++ b/migrations/migration-runner.js
@@ -18,11 +18,4 @@ setUpServer();
 
 // Replace this with your migration
 const processUsers = require('@migrationScript@');
-processUsers()
-  .then(function success () {
-    process.exitCode = 0;
-  })
-  .catch(function failure (err) {
-    console.log(err);
-    process.exitCode = 1;
-  });
+processUsers();
